source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -67,96 +67,95 @@ Steps
       The following API keys are found in the Confluent Cloud console. For more 
       information, see `Use API Keys <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
 
-      .. procedure::
 
-         .. step:: Configure Confluent Cloud API keys
 
-            In the top right corner of the Confluent Console, click the :icon-fa5:`bars` icon. 
-            Under :guilabel:`Administration`, select :guilabel:`Cloud API Keys`.
+      a) Configure Confluent Cloud API keys
 
-            Create Confluent Cloud API keys and configure permissions for them. You 
-            have two permission options:
+         In the top right corner of the Confluent Console, click the :icon-fa5:`bars` icon. 
+         Under :guilabel:`Administration`, select :guilabel:`Cloud API Keys`.
 
-            1. Grant global access.
+         Create Confluent Cloud API keys and configure permissions for them. You 
+         have two permission options:
```

Comment:
[question] Should this be an unordered list?

Reply:
@mdb-ashley I feel like there's no reason for it to have numbers, I'll change to bullet points